source/access/managing_disk_usage.rst

```diff
@@ -1,105 +1,138 @@
 .. _disk usage:
 
-Managing disk usage
-===================
+How much disk space am I using?
+===============================
 
-Total disk space used on filesystems with quota      
+.. _quota:
+
+Total disk space used on filesystems with quota
 -----------------------------------------------
-                                  
-On filesystems with 'quota enabled', you can check the amount of disk space that
-is available for you, and the amount of disk space that is in use by   
-you. Unfortunately, there is not a single command that will give   
-you that information for all file systems in the VSC.               
-                                  
--  ``quota`` is the standard command to request your disk   
-   quota. Its output is in 'blocks', but can also be      
-   given in MB/GB if you use the ``-s`` option.                   
--  But it does not work on GPFS file systems. On those you     
-   have to use ``mmlsquota``.  This is the case for the       
-   scratch space at the KU Leuven or on the Tier-1.              
--  On some clusters, these commands are currently disabled.
--  Also, using these commands on another cluster than the one   
-   in your home institution, will fail to return information     
-   about the quota on your VSC_HOME and VSC_DATA          
-   directories and will show you  the quota for your VSC_SCRATCH 
-   directory on that system.      
-                                  
-::                                
-                                  
+
+On filesystems with quota enabled, you can check the amount of disk space that
+is available for you, and the amount of disk space that is in use by
+you.  Unfortunately, there is not a single command that will give
+you that information for all file systems in the VSC.
+
+On most systems, ``quota`` will show you for each file system
```

Comment:
is it really "most"? this does not work for gpfs, and thus in gent does nothing useful (and afaik, also in brussel @wpoely86). 
for gent (and brussel), users can check their usage and limits in the AP

Reply:
It seems to work for $VSC_HOME and $VSC_DATA on all sites, so that would be "most".

Reply:
it works for kuleuven users everywhere, sure, because it's nfs. but it doesn't work for e.g. gent users 
anyway, my main point is that i don't mind that this is documented, but this is rather kuleuven specific, and that is not very clear from some parts of the text